README.md

```diff
@@ -138,3 +138,22 @@ and under the GitHub settings add
 ```
 where `owner` is the GitHub user/org,
 and `repository` is the name of the repository you want to open.
+
+## Development
+
+For a development install, do the following in the repository directory:
+
+```bash
+jlpm install
+jlpm run build
+jupyter labextension link .
+```
+
+You can then run JupyterLab in developer mode to automatically pick up changes
+to `@jupyterlab/github`. Launch JupyterLab using
+```bash
+jupyter lab --watch
+```
+This will automatically recompile `@jupyterlab/github` upon changes, and
```

Comment:
The `jupyter lab --watch` process won't automatically recompile the extension typescript sources, it instead watches the built JS. So you still need to watch or recompile the extension for JupyterLab to pick up the changes.

Reply:
Thanks for the review, @ian-r-rose! What would you recommend then? `jupyter lab --watch --dev-mode`? Or another step for recompilation?

Reply:
Not `--dev-mode` (which will run JupyterLab from master without extensions).  Instead we can start a watch process in the `jupyterlab-github` directory (`jlpm run watch`), which will recompile the extension *at the same time* as the `jupyter lab --watch` process is going on, which will then pick up the changes to the compiled sources.